From: Mathieu Parent <math.parent@gmail.com>
Date: Fri, 31 Jan 2014 12:11:25 +0100
Subject: LSB: Update ref url

---
 bin/init.d/shinken | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/bin/init.d/shinken b/bin/init.d/shinken
index 4951d9a..ed45652 100755
--- a/bin/init.d/shinken
+++ b/bin/init.d/shinken
@@ -37,9 +16,10 @@
 #     - receiver    : manage all passive data
 ### END INIT INFO

-# Deadlink :(
-# http://refspecs.freestandards.org/LSB_3.1.1/LSB-Core-generic/LSB-Core-generic/iniscrptact.html
+# Reference:
+# http://refspecs.linuxfoundation.org/LSB_4.1.0/LSB-Core-generic/LSB-Core-generic/iniscrptact.html

+. /lib/lsb/init-functions

 NAME="shinken"
