diff --git a/bin/shinken b/bin/shinken
index 9488112..662ea19 100755
--- a/bin/shinken
+++ b/bin/shinken
@@ -119,7 +119,7 @@ class CLICommander(object):
 # the Linux Standard Base''',
             'values'  : [('etc', '/etc/shinken'), ('lib','/var/lib/shinken'),
                          ('share', '%(lib)s/share'), ('cli','%(lib)s/cli'),
-                         ('packs', '%(etc)s/packs'), ('modules', '%(lib)s/modules'),
+                         ('packs', '%(etc)s/packs'), ('modules', '/usr/share/pyshared/shinken/modules'),
                          ('doc', '%(lib)s/doc'), ('inventory', '%(lib)s/inventory'),
                          ('libexec', '%(lib)s/libexec'),
                          ]}
diff --git a/cli/desc/cli.py b/cli/desc/cli.py
index fc12c12..514023a 100644
--- a/cli/desc/cli.py
+++ b/cli/desc/cli.py
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
-
 # Copyright (C) 2009-2012:
 #    Gabes Jean, naparuba@gmail.com
 #    Gerhard Lausser, Gerhard.Lausser@consol.de
diff --git a/cli/doc/cli.py b/cli/doc/cli.py
index 17df734..4fdcf94 100644
--- a/cli/doc/cli.py
+++ b/cli/doc/cli.py
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
-
 # Copyright (C) 2009-2012:
 #    Gabes Jean, naparuba@gmail.com
 #    Gerhard Lausser, Gerhard.Lausser@consol.de
diff --git a/cli/shinkenio/cli.py b/cli/shinkenio/cli.py
index b3dcf0d..c14de14 100644
--- a/cli/shinkenio/cli.py
+++ b/cli/shinkenio/cli.py
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
-
 # Copyright (C) 2009-2014:
 #    Gabes Jean, naparuba@gmail.com
 #    Gerhard Lausser, Gerhard.Lausser@consol.de
